ivadomed/scripts/automate_training.py

```diff
@@ -138,24 +138,27 @@ def automate_training(fname_config, fname_param, fixed_split, all_combinations,
     # TODO: add example of DF
 
     Args:
-        fname_config (string): Configuration filename, which is used as skeleton to configure the training. Some of its
-            parameters (defined in `fname_param` file) are modified across experiments.
-        fname_param (string): json file containing parameters configurations to compare. Parameter "keys" of this file
-            need to match the parameter "keys" of `fname_config` file. Parameter "values" are in a list. Example::
+        config (string): Configuration filename, which is used as skeleton to configure the training. Some of its
+            parameters (defined in `param` file) are modified across experiments. Flag: --config, -c
+        param (string): json file containing parameters configurations to compare. Parameter "keys" of this file
+            need to match the parameter "keys" of `config` file. Parameter "values" are in a list. Flag: --param, -p
+             Example::
```

Comment:
We have few other functions whose arguments are `fname_*`: I don't know what is best, ie * or fname_*.
What was your motivation here?

Reply:
Hum match that with the flag as discussed on Monday (issue #329)

Reply:
My question was: you could have go the other way ;) ie fname_* everywhere
But really that's a detail. Happy either way. I was mostly curious to know if there were a reason.

Reply:
Ah sorry! I wanted to keep the flag (fname_config and fname_param would have been -f)